terraform/files/kubernetes-manifests.d/nginx-ingress/base/nginx-ingress-controller-v1.8.0.yaml

```diff
@@ -14,28 +14,21 @@ metadata:
   labels:
     app.kubernetes.io/component: controller
     app.kubernetes.io/instance: ingress-nginx
-    app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/name: ingress-nginx
     app.kubernetes.io/part-of: ingress-nginx
-    app.kubernetes.io/version: 1.1.2
-    helm.sh/chart: ingress-nginx-4.0.18
+    app.kubernetes.io/version: 1.8.0
```

Comment:
I think it is not a fix for #440 but mainly for #276

Reply:
But as far as i know, it doesn't change the functionality, because it is downloaded [here](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/blob/main/terraform/files/bin/apply_nginx_ingress.sh#L18)

Reply:
Ok i removed it, see caa7c47cc00c8cd2626094810b9e097c8834f3d5